src/Evolution/Systems/ScalarAdvection/Subcell/TciOnFdGrid.cpp

```diff
@@ -0,0 +1,33 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/ScalarAdvection/Subcell/TciOnFdGrid.hpp"
+
+#include <cstddef>
+
+#include "Evolution/DgSubcell/PerssonTci.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+namespace ScalarAdvection::subcell {
+template <size_t Dim>
+bool TciOnFdGrid<Dim>::apply(const Scalar<DataVector>& dg_u,
+                             const Mesh<Dim>& dg_mesh,
+                             const double persson_exponent) noexcept {
+  constexpr double persson_tci_epsilon = 1.0e-18;
+  // In order to prevent rapid switching between the DG and FD schemes, we use
+  // persson exponent += 1 for the TCI when deciding whether to switch back to
+  // DG.
+  return ::evolution::dg::subcell::persson_tci(
+      dg_u, dg_mesh, persson_exponent + 1.0, persson_tci_epsilon);
```

Comment:
@nilsdeppe As I understood from the paper, it seems that the `persson_exponent` needs to be +=1 here before switching to DG from FD (from the statements in the comments above).. is that correct?

Reply:
The correct exponent is passed into the function, you don't need to change it here. That is, it's passed in with the `+1` included

Reply:
Yep, I found the action doing it. Fixed :)